README.md

```diff
@@ -1,153 +1,170 @@
+central location to simplify monitoring and administration of the cluster.
  
 ###Node
-Node lifecycle management requires handling the following:
-- **Bootstrap**: It should be possible to bootstrap the nodes remotely. *As described above this
-feature is not provided in initial release of the cluster manager.*
-- **Cluster membership**: It should be possible to automatically track cluster's membership of
-reachable and unreachable nodes.
-- **Commission**: It should be possible for the operator to remotely commission a reachable node.
-Optionally it should be possible to automatically commission newly discovered node. This shall be
-configurable in cluster manager configuration parameter.
-- **Upgrade**: It should be possible for the operator to remotely upgrade a reachable node.
-Optionally it should be possible to automatically upgrade nodes when configuration repository
-changes. This shall be configurable in cluster manager configuration parameter.
-- **Decommission**: It should be possible for the operator to remotely decommission a node.
-- **Batch operations**: It should be possible for the operator to commission, upgrade or decommission
-all or a batch of nodes.
-- **Reloads**: In event of node reloads (or node reachability changes) it should be possible to verify the
-node's configuration and automate corrective actions in case of failures.
+Node lifecycle management provides:
+- **Bootstrap**: Remotely provision a node's image from a central location. *As described
+above this feature is not provided in initial release of the Contiv Cluster.*
+- **Cluster Membership**: Automatically track cluster's membership of reachable and unreachable nodes.
+- **Commission**: Remotely provision a node's clustering software, configuration, etc..
+Optionally commission newly discovered nodes automatically.
+- **Upgrade**: Remotely upgrade the image or a software component of a commissioned node.
+Optionally upgrade nodes when a configuration repository changes.
+- **Decommission**: Automatically remove configuration, software components, etcd from a commissioned node.
+- **Batch operations**: Commission, upgrade or decommission all or a subset number of nodes.
+- **Reloads**: In event event of a node reload or unreachability event, verify the node configuration
+and automate corrective actions.
 
 ###Cluster
-- **Bootstrap**: It should be possible to bootstrap the cluster remotely. *As described above this
-feature is provided in form a bootstrap script that will need to be manually run on the first node
-in initial release of the cluster manager.*
-- **Sustenance and Availability**: Cluster manager service shall remain available as long as there
-is one node up and running in the cluster.
+The following Cluster management capabilities are provided by Contiv Cluster:
+- **Bootstrap**: Remotely bootstrapping a cluster. *For the initial release of Contiv Cluster,
```

Comment:
this can be removed as well now:

> *For the initial release of Contiv Cluster, this feature is provided as a bootstrap script that is run manually on the first node.*



Also 
> Remotely bootstrapping a cluster

can just be 

```
bootstrap a cluster starting with one or more nodes
```